llvm/test/CodeGen/X86/isel-fabs.ll

```diff
@@ -0,0 +1,52 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc < %s  -mtriple=x86_64-unknown-linux-gnu -global-isel=0 -mattr=+x87,+sse,+sse2 -o - | FileCheck %s --check-prefix=X64
+; RUN: llc < %s -mtriple=x86_64-unknown-linux-gnu -global-isel=1 -mattr=+x87,+sse,+sse2 -o - | FileCheck %s -check-prefixes=GISEL
```

Comment:
Can we add a common prefix for SDAG and GISEL? Usually we use `X64` or `X86` as a common prefix and something like `X64-SDAG/GISEL` as unique ones.